src/types.ts

```diff
@@ -10,8 +10,19 @@ export interface RedisOptions {
     key?: string
     secret?: string
   }
+  connectTimeout?: number
+  reconnectOnError?: IoredisReconnectOnErrorStrategy
 }
 
+/**
+ * - https://github.com/redis/ioredis/tree/v5.4.1?tab=readme-ov-file#reconnect-on-error
+ * - https://github.com/redis/ioredis/blob/v5.4.1/lib/redis/RedisOptions.ts#L78
+ */
+export type IoredisReconnectOnErrorStrategy =
```

Comment:
I'm not too happy with the number config here. I know it comes from Ioredis, but it's not very self-explanatory. :)

A better solution could be to have to boolean options instead:
- `reconnectOnError: boolean`
- `resendOnReconnect: boolean`

You already have a `mapToReconnectOnError()` function where this could happen.

We could also have an enum with the three options, but I think two booleans is still easier to understand. Agree?

Reply:
👍 I'll fix that as soon as possible! But another thin just came up here, so I'll have to wait a bit.

Reply:
And thanks for the quick review 👍